airbyte-commons/src/main/java/io/airbyte/commons/text/Names.java

```diff
@@ -43,4 +43,12 @@ public static String toAlphanumericAndUnderscore(String s) {
         .replaceAll(NON_ALPHANUMERIC_AND_UNDERSCORE_PATTERN, "_");
   }
 
+  public static String concatNames(String name, String s) {
```

Comment:
This function is oddly specific with a non descriptive name, yet in the commons library. 

So, either you call it concatQuotedNames and its behavior is quote(unquote(name) + unquote(s)) or it shouldn't in this file.

can you add a test?